### DISCUSSION TITLE
[Select] Is there a way to limit the number of options shown at once？

### DISCUSSION THREAD

**Discussion by hhhhzs** [UI5/webcomponents#6242] [Q&A]
Hi everyone,
When there are a lot of options in the selector, the option list would fill in all the space and show as many options as it can. Is there any way to limit the number of options shown at once, for example, show only 5 options at most, to make the layout prettier? 
Thanks
<img width="247" alt="image" src="https://user-images.githubusercontent.com/57512939/210699492-419ed28f-e601-4c16-849d-b4782ad0ab70.png">

**Comment by Lukas742** [comment]
Hi @hhhhzs 

I'm not aware of any API to support this behavior and since this component is developed by the UI5 Web Components team, I'm going to transfer this question to their repo.
